nixos/modules/security/wrappers/default.nix

```diff
@@ -92,7 +90,7 @@ let
     }:
     ''
       cp ${securityWrapper}/bin/security-wrapper "$wrapperDir/${program}"
```

Comment:
What do you think of `nix-setuid-wrapper` as a name for the executable? And then prefix all the error messages in the program with it, so it's clear that they come from this pretty unique name if they are issued.

Reply:
I think this was renamed from setuid wrapper to security wrapper, because there is setuid, setgid, setcap, and who knows what gets added later…

But maybe `nixos-security-wrapper` ?

Reply:
renamed.